pkg/scheduler/internal/heap/heap.go

```diff
@@ -158,6 +164,14 @@ func (h *Heap) Update(obj interface{}) error {
 	return h.Add(obj)
 }
 
+// Cleanup removes all items.
+func (h *Heap) Cleanup() {
```

Comment:
Why are you adding this function?

Reply:
The `Clear` function of the metric recorder is never called in `heap.go`. Should I remove it ?

Reply:
It's called by `UnschedulablePods`. Just remove this function if it's not needed.

Reply:
Got it. I removed it. sorry